civihr_employee_portal/civihr_employee_portal.install

```diff
@@ -91,3 +91,16 @@ function civihr_employee_portal_update_7005() {
   $schema = module_invoke('civihr_employee_portal', 'schema');
   db_create_table('reports_configuration', $schema['reports_configuration']);
 }
+
+/**
+ * Create civihr_admin_local Role for existing sites.
```

Comment:
the problem of drupal upgraders that they require you to update all the modules too (right ? ) so I just feel this is not need , what do you think ?